.pre-commit-config.yaml

```diff
@@ -22,6 +22,13 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
+        args: ["--profile", "black", "--filter-files"]
+
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        args: ["-l", "119"]
```

Comment:
I think we can add Python's version after this line, as suggested in the documentation. Check [black's docs](https://black.readthedocs.io/en/stable/integrations/source_version_control.html).

`language_version: python3.10`

Reply:
Either is fine, the choice is yours. :)

About the separate PR, no need. We're already reviewing the pre-commit hooks, starting with black. You can change the title of the PR if you want, but not necessary IMO.

Reply:
Done. #Resolved?